sdk/common/src/test/java/io/opentelemetry/sdk/resources/ResourceTest.java

```diff
@@ -154,15 +167,15 @@ void testMergeResources() {
         Attributes.of(stringKey("a"), "1", stringKey("b"), "2", stringKey("c"), "4");
 
     Resource resource = DEFAULT_RESOURCE.merge(resource1).merge(resource2);
-    assertThat(resource.getAttributes()).isEqualTo(expectedAttributes);
+    assertThat(resource.getAttributes().asMap()).containsAllEntriesOf(expectedAttributes.asMap());
```

Comment:
Why changing the previous call way of checking equality? Same question everywhere.

Reply:
Mostly because if I'm on Windows, the fallback name is different than on other systems, and doing it as a map let's me compare that it at least has the right stuff in it.

Reply:
the other approach, I guess, would be to expose the fallback value so tests could access it and put it into the expected. Not sure if that's better or worse, TBH.